content/es/about-us.html

```diff
@@ -0,0 +1,22 @@
++++
+title = "Acerca de Vote.gov"
+aliases = [
+  "/about"
```

Comment:
Please change the name of this file to `acerca-de-votegov`.
Additionally you can remove the alias, this was for English only.

Reply:
Please also update the footer link in this translation to point to this page and remove the _(in english)_ part.